vrt/tests.vrt.js

```diff
@@ -20,8 +20,13 @@ const toMatchImageSnapshot = configureToMatchImageSnapshot({
 
 expect.extend({toMatchImageSnapshot});
 
+const allScenarios = [
+  ...getAllScenarioNames(),
+  ...getAllScenarioNames().map(scenarioName => `${scenarioName}-dark`),
```

Comment:
how long do vrts take? do we need to snapshot all mobile screens in dark-mode as well?

Reply:
yeah, I am not sure yet - we can update once we have the first run from the CI